lib-core/coffee/ks.coffee

```diff
@@ -1,7 +1,7 @@
 global.k$ = new Object()
 
 k$.$$ = (el) -> document.querySelectorAll el
-k$.$ = (el) -> k$.$$(el)[0]
```

Comment:
querySelector() is presumably more efficient than querySelectorAll()[0] because it can stop after finding a single match whereas querySelectorAll will continue to process the DOM to find all the matches.